pyqtgraph/graphicsItems/ViewBox/ViewBoxMenu.py

```diff
@@ -238,7 +237,7 @@ def setViewList(self, views):
             
         for i in [0,1]:
             c = self.ctrl[i].linkCombo
-            current = asUnicode(c.currentText())
+            current = str(c.currentText())
```

Comment:
since `c` is a `QComboBox` we can be sure that `QComboBox.currentText()` returns a `str` so we can ditch this casting.